github/ci/prow-deploy/kustom/base/manifests/test_infra/current/statusreconciler_rbac.yaml

```diff
@@ -17,19 +17,6 @@ rules:
     verbs:
       - create
 ---
-kind: Role
```

Comment:
/hold 

@dhiller - this looks to be removing the role and rolebinding that you added for the statusreconciler?

Reply:
Yeah, maybe we need to do this as a patch...

Reply:
Should be fixed with https://github.com/kubevirt/project-infra/pull/3196